config/config/common-config.yaml

```diff
@@ -355,7 +355,7 @@ kubeStateMetrics:
     limits:
       cpu: 30m
       memory: 100Mi
-  # Enable this if your cluster is managed by Cluster API to get Cluster API metrics.
+  # Configure this if DNS-01 challenges are enabled in cert-manager
```

Comment:
Should the old comment not also be here? It's also relevant right?

Reply:
This should not have been overwritten :eyes: I will fix it, thanks for noticing :+1: